sklearn/preprocessing/__init__.py

```diff
@@ -7,23 +7,24 @@
 
 from ._data import Binarizer
 from ._data import KernelCenterer
-from ._data import MinMaxScaler
 from ._data import MaxAbsScaler
+from ._data import MinMaxScaler
```

Comment:
Is your editor using `isort` here? :)

Reply:
It seems I'm more pedantic than my editor :blush:.

Reply:
Please avoid and revert these @lorentzenchr, and any other change that is not directly related to the PR (like removal of spaces, license, or doc formatting), since they distract from the review. They also unnecessarily obfuscate git blame.

It might not seem like much but it does take time and attention from reviewers at each review, especially when reviews are spread out (which they often are)

As a bonus reviewers are less scared to even start reviewing since the displayed number of changes is smaller ;)